cmd/url.go

```diff
@@ -2,10 +2,12 @@ package cmd
 
 import (
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/odo/util"
 	"os"
 	"strings"
 
+	"github.com/redhat-developer/odo/pkg/odo/util"
+	util2 "github.com/redhat-developer/odo/pkg/util"
```

Comment:
why util2? should we have this renamed instead? ;)

Reply:
what? Why do we have two utils??
@metacosm Do I understand correctly that `github.com/redhat-developer/odo/pkg/odo/util`  is  utils for code around cmd?  If yes, than it might be good to import it as `cmdutils` or something like that, @cdrage. We have to have to have better names than just `util` and `util2`, that will be mess.

Reply:
Yeah, I was taking this inspiration from @geoand 's PR :) I'll rename it